GUI/URLHandlers.cs

```diff
@@ -13,17 +13,26 @@ public static class URLHandlers
         private static readonly ILog log = LogManager.GetLogger(typeof(URLHandlers));
         public const string UrlRegistrationArgument = "registerUrl";
 
-        private static string MimeAppsListPath = ".local/share/applications/mimeapps.list";
+        private static string MimeAppsListPath = "mimeapps.list";
         private static string ApplicationsPath = ".local/share/applications/";
         private const string HandlerFileName = "ckan-handler.desktop";
 
         static URLHandlers()
         {
             if (Platform.IsUnix)
             {
-                string home = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
-                MimeAppsListPath = Path.Combine(home, MimeAppsListPath);
-                ApplicationsPath = Path.Combine(home, ApplicationsPath);
+                string XDGDataHome = Environment.GetEnvironmentVariable("XDG_DATA_HOME");
+                if (XDGDataHome != null)
+                {
+                    ApplicationsPath = Path.Combine(XDGDataHome, "applications");
```

Comment:
The `applications` subdirectory is deprecated. The preferred path is [specced here](https://standards.freedesktop.org/mime-apps-spec/mime-apps-spec-1.0.1.html)